src/nimvscode.nim

```diff
@@ -292,6 +296,46 @@ proc clearCachesCmd(): void =
   let config = vscode.workspace.getConfiguration("files")
   discard clearCaches(config.getStrBoolMap("watcherExclude", defaultIndexExcludeGlobs))
 
+proc initializeClient(nimls: cstring) =
```

Comment:
Please put these in their own module, then export/import the minimum

Reply:
sure.